src/bidiMapper/domains/script/scriptEvaluator.ts

```diff
@@ -17,16 +17,74 @@
 import {Protocol} from 'devtools-protocol';
 
 import {CommonDataTypes, Message, Script} from '../../../protocol/protocol.js';
+import {IEventManager} from '../events/EventManager.js';
 
 import {Realm} from './realm.js';
 
+async function longPoll(
```

Comment:
I welcome suggestions for a better name for this method, instead of `longPoll`.

Reply:
`initChannelListener`?

Reply:
Done